crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml

```diff
@@ -78,6 +78,8 @@
                 %span.help-block
                   = t(".volumes.netapp.nfs_shares_config_hint")
 
+              = string_field %w(volumes {{@index}} netapp nfs_mount_options)
```

Comment:
You need to add one indentation level so that it's inside the nfs_container div.